File: 2679-sum-in-a-matrix/2679-sum-in-a-matrix.py
class Solution:
    def matrixSum(self, nums: List[List[int]]) -> int:
        nums, res = [sorted(row) for row in nums], 0
        for i in range(0, len(nums[0])):
            mx = float("-inf")
            for row in nums:
                mx = max(mx, row[i])
            res += mx
        return res

File: 2679-sum-in-a-matrix/2679-sum-in-a-matrix.js
/**
 * @param {number[][]} nums
 * @return {number}
 */
var matrixSum = function(nums) {
    nums.forEach(row => row.sort((a, b) => b - a))
    let res = 0
    for (let i=0; i < nums[0].length; i++) {
        let max = -Infinity
        for (const row of nums) {
            max = Math.max(max, row[i])
        }
        res += max
    }
    return res
};